app/controllers/api/v1/media_controller.rb

```diff
@@ -25,7 +25,7 @@ class MediaController < V1::BaseController
 
       api :GET, "/media/", "List all media."
       param :search, String, :desc => "filter results", :required => false
-      param :order, String, :desc => "sort results", :required => false, :desc => "for example, name ASC, or name DESC"
+      param :order, String, :required => false, :desc => "for example, name ASC, or name DESC"
```

Comment:
"sort results" is the usual description, please delete the other one.  ditto below